student.rb

```diff
@@ -0,0 +1,11 @@
+class Student < Person
+  attattr_accessor :@classroom
+  def initialize(classroom, age, name = 'Unknown', parent_permission: true)
+    @classroom = classroom
+    super(age, name, parent_permission: parent_permission)
```

Comment:
[OPTIONAL]
- [ ] It would be better to have the inherited variables i.e `super` before the instance variable `@classroom`
- [ ] Also in the `super` it should be 
```rb
def initialize(classroom, age, name = 'Unknown', parent_permission = true)
    @classroom = classroom
    super(age, name, parent_permission)
```